frontend/src/views/web/challenge/submission.html

```diff
@@ -351,22 +364,69 @@ <h5 class="w-300">Make Submission</h5>
+                                </section>
+                            </div>
+
+                            <!-- Submission using cli-->
+                            <div ng-show="challenge.isSubmissionUsingCli === true">
+                                <li><p class="fs-18 w-400">Submission instructions</p></li>
+                                <div class="fs-16 w-300">
+                                    <li>
+                                        1. Install evalai-cli<br>
+                                        <code>$ pip install evalai{{challenge.cliVersion}}</code>
+                                        <button class="btn ev-btn-dark waves-effect waves-dark grad-btn code-btn"
+                                            ng-click="challenge.showConfirmation('Command copied to clipboard')" ngclipboard
+                                            data-clipboard-text='pip install "evalai{{challenge.cliVersion}}"'>
+                                            <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
+                                        </button>
+                                    </li>
+                                    <li>
+                                        2. Add your EvalAI account token to evalai-cli<br>
+                                        <code>$ evalai set_token {{challenge.authToken}}</code>
+                                        <button class="btn ev-btn-dark waves-effect waves-dark grad-btn code-btn"
+                                            ng-click="challenge.showConfirmation('Command copied to clipboard')" ngclipboard
+                                            data-clipboard-text="evalai set_token {{challenge.authToken}}">
+                                            <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
+                                        </button>
+                                    </li>
+                                    <li>
+                                        3. Make submission<br>
+                                        <code>$ evalai challenge {{challenge.challengeId}} phase {{challenge.phaseId}} submit &lt;submission_file_path&gt; --large</code>
+                                        <button class="btn ev-btn-dark waves-effect waves-dark grad-btn code-btn"
+                                            ng-click="challenge.showConfirmation('Command copied to clipboard')" ngclipboard
+                                            data-clipboard-text="evalai challenge {{challenge.challengeId}} phase {{challenge.phaseId}} submit &lt;submission_file_path&gt; --large">
+                                            <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
+                                        </button>
+                                    </li>
+                                    <li>
+                                        4. Make private submission<br>
```

Comment:
Use --private or --public flag in the submission command to make the submission private or public respectively.